tests/unit/routers/test_platform.py

```diff
@@ -67,34 +71,44 @@ def test_read_heartbeat(anon_client):
         resp = anon_client.get("/__heartbeat__")
     assert resp.status_code == 200
     data = resp.json()
-    expected = {"database": {"up": True, "time_ms": data["database"]["time_ms"]}}
-    assert data == expected
+    assert data == {
+        "checks": {"database": "ok"},
+        "details": {},
+        "status": "ok",
+    }
     assert len(cap_logs) == 1
 
 
-def test_read_heartbeat_no_db_fails(anon_client, mock_db):
+def test_read_heartbeat_db_fails(anon_client, mock_db):
     """/__heartbeat__ returns 503 when the database is unavailable."""
-    mock_db.execute.side_effect = SQATimeoutError()
+    next(mock_db).execute.side_effect = SQATimeoutError()
```

Comment:
Related to my comment above, but if you use `SessionLocal` directly in the Dockerflow check, I found this was a way to provide the mock here:

```python
@patch("ctms.routers.platform.SessionLocal")
def test_read_heartbeat_db_fails(mock_db, anon_client):
    """/__heartbeat__ returns 503 when the database is unavailable."""

    mocked_session = MagicMock()
    mocked_session.__enter__.return_value = mocked_session
    mocked_session.execute.side_effect = SQATimeoutError()
    mock_db.return_value = mocked_session
```

Also, directly in the test and not in a fixture since it's only used once.